snmp/tests/test_unit.py

```diff
@@ -91,7 +91,7 @@ def test_parse_metrics(lcd_mock):
 
     # Table with manual OID
     metrics = [{"MIB": "foo_mib", "table": "foo_table", "symbols": [{"OID": "1.2.3", "name": "foo"}]}]
-    oids, _, parsed_metrics = config.parse_metrics(metrics)
+    _, oids, _, parsed_metrics = config.parse_metrics(metrics)
```

Comment:
We should prob use the name `next_oids` here, because that's what's returned, correct? I agree it'd result in a bigger diff but it's more correct.